dashboard/test/models/pd/async_workshop_handler_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+require 'fake_sqs/test_integration'
+
+# Launch a fake SQS service running on Localhost
+Aws.config.update(region: 'us-east-1', access_key_id: 'fake id', secret_access_key: 'fake secret')
+$fake_sqs_service = FakeSQS::TestIntegration.new(database: ':memory:',
+  sqs_endpoint: 'localhost', sqs_port: 4568)
+sleep(2) # add a sleep to fix test failures with 'RuntimeError: FakeSQS didn't start in time'
```

Comment:
I copied it from [ActivityTest](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/models/activity_test.rb#L14). It looks like FakeSQS [start](https://github.com/iain/fake_sqs/blob/7d6cb0504d96abf7eab9128a3bcb421b04dcd498/lib/fake_sqs/test_integration.rb#L28) should handle that for us, but I'm a little suspect given that we have this in the other test. I'd like to leave this in for now to be safe.